--- nano-2.2.6.orig/configure.ac	2012-03-07 19:13:30.196063086 -0800
+++ nano-2.2.6/configure.ac	2012-03-07 19:07:46.876071285 -0800
@@ -442,7 +442,7 @@
     AC_CHECK_HEADERS(ncurses.h)
 
     if test x$enable_utf8 != xno; then
-	AC_CHECK_LIB(ncursesw, get_wch, [CURSES_LIB="-lncursesw" CPPFLAGS="-I/usr/include/ncursesw $CPPFLAGS" CURSES_LIB_NAME=ncursesw CURSES_LIB_WIDE=yes])
+	AC_CHECK_LIB(ncursesw, get_wch, [CURSES_LIB="-lncursesw" CURSES_LIB_NAME=ncursesw CURSES_LIB_WIDE=yes])
     fi
 
     if eval "test x$CURSES_LIB_NAME = x"; then

